cog/daily_charge.py

```
@@ -23,95 +23,241 @@
+        charge_channel_id = get_channels()["everyDayCharge"]
+        messages = await get_history(
+            self.bot,
+            charge_channel_id,
+            after=earliest_downtime_start,
+            before=latest_downtime_end,
+        )
+
+        # TODO: cache the last_charged date
+        # cached_change: UserDict = UserDict()
+        with mysql_connection() as cursor:
+            assert (
```

Ruff (0.8.2)

Use of `assert` detected

(S101)